topic-controller/src/test/java/io/strimzi/controller/topic/ControllerTest.java

```diff
@@ -52,23 +54,22 @@
 
     private final LabelPredicate cmPredicate = new LabelPredicate(
             "kind", "topic",
-            "app", "barnabas");
+            "app", "strimzi");
```

Comment:
How important it would be to run the tests with the labels which we use (i.e. domain prefixed)? Can it catch some more issues? Or is that something what doesn't matter?

Reply:
While `strimzi.io/kind=topic` is the default it can be changed, so I don't see any harm in testing with other labels.

Reply:
Ok, good.